packages/aws-cdk-lib/aws-lambda/README.md

```diff
@@ -902,6 +902,23 @@ const fn = new lambda.Function(this, 'MyFunction', {
 See [the AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/concurrent-executions.html)
 managing concurrency.
 
+## Lambda with SnapStart
+
+SnapStart is currently supported only on Java 11/Java 17 runtime. SnapStart does not support provisioned concurrency, the arm64 architecture, Amazon Elastic File System (Amazon EFS), or ephemeral storage greater than 512 MB. After you enable Lambda SnapStart for a particular Lambda function, publishing a new version of the function will trigger an optimization process.
+
+See [the AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/snapstart.html) to learn more about AWS Lambda SnapStart
+
+```ts
+const fn = new lambda.Function(this, 'MyFunction', {
+  code: lambda.Code.fromAsset(path.join(__dirname, 'handler.zip')),
+  runtime: lambda.Runtime.JAVA_11,
+  handler: 'example.Handler::handleRequest',
+  snapStart: lambda.SnapStartConfig.ON_PUBLISHED_VERSIONS,
```

Comment:
@roger-zhangg in case you didn't catch this from the build logs, it looks like the build is failing because you changed the class to `SnapStartConf`, but didn't update it here.

Reply:
Thanks for the catch, I'll update it

Reply:
Also I wonder is there a local equivalent to this cloud build? So I won't miss next time

Reply:
No that I'm aware of. If I have a failing build I always use the build logs we generate to understand why it's failing. In this case, at the bottom of the build logs it pointed to the `README` so I knew to check there.